src/utils/uniqueKey_generator.js

```diff
@@ -0,0 +1,12 @@
+export default function createUniqueKey() {
+  let dt = new Date().getTime();
+  const tempStr = 'xxxxxx.yxyxxyxy.xxxyyy';
+  const uuid = tempStr.replace(/[xy]/g, function(c) {
+    /* eslint no-bitwise: 0 */
+    const r = (dt + Math.random() * 16) % 16 | 0;
+    dt = Math.floor(dt / 16);
+    /* eslint no-bitwise: 0 */
+    return (c === 'x' ? r : (r & 0x3) | 0x8).toString(16);
+  });
```

Comment:
Does line 7 actually accomplish anything?  (It looks like its result is never used.). At line 6, what is the effect of adding `Math.random() * 16` if you're going to do a modulo 16 operation on the sum?  Isn't the result the same as `dt % 16`??  Were you intending to do the division before the modulo?

What does the `| 0` at line 6 do?  (I'm sure it does something wonderfully clever...I just can't remember what...does it give you an unsigned result?  Is it ensuring that you have a 32-bit result?)

Since you're already doing bitwise operations (and since 16 is a magic number), I recommend replacing the modulo operator in line 6 with a bitmask:
```
    const r = (dt + Math.random() * 16) & 0xf | 0;
```

Regarding the ESLint directives, it looks like you have [better options](https://eslint.org/docs/rules/no-bitwise) than just disabling it.
```
    /*eslint no-bitwise: ["error", { "int32Hint": true }] */
    const r = (dt + Math.random() * 16) % 16 | 0;
    /*eslint no-bitwise: ["error", { "allow": ["&", "|"] }] */
    return (c === 'x' ? r : (r & 0x3) | 0x8).toString(16);
```

Reply:
Also, this code is triggering an LGTM warning, which I think you could consider suppressing, by adding this to the end of line 6:
```
  // lgtm[js/insecure-randomness]
```

Reply:
Removed line 7 (left it by mistake), Added correct ESLint directives.